### DISCUSSION TITLE
student benefits when do I receive them?

### DISCUSSION THREAD

**Discussion by ramonlejandro0226** [community/community#132391] [GitHub Education]
### Hi everyone,

Hello everyone, I just received my student benefits approval. I would like to know when I will receive the benefits.

**Comment by ghost** [comment]
Hi, the benefits mostly appear after a few days and sometimes after a week.

**Comment by ramonlejandro0226** [comment]
Oh thanks, but how do I know I'm getting the benefits? and you are a student or part of the github staff

**Reply by ghost** [reply]
you will receive a message or notification that benefits are available. I am a regular user, staff have a blue icon labeled “staff”.
